man/systemd-repart.xml

```diff
@@ -578,6 +578,18 @@
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--generate-tabs=</option><replaceable>BOOL</replaceable></term>
```

Comment:
shouldn't this rather be: `--generate-fstab=/path/to/fstab` and `--generate-crypttab=/path/to/crypttab`?